galera/map.jinja

```diff
@@ -1,7 +1,7 @@
 
 {%- set master = salt['grains.filter_by']({
   'Debian': {
-    'pkgs': ['mysql-server-wsrep', 'galera', 'rsync', 'python-mysqldb', 'libmysqlclient18', 'psmisc', 'netcat', 'percona-xtrabackup-21', 'socat'],
+    'pkgs': ['mariadb-server-10.1', 'galera-3', 'libmariadbclient18', 'libmysqlclient18', 'mariadb-client-10.1', 'mariadb-client-core-10.1', 'mariadb-common', 'mariadb-server-core-10.1', 'mysql-common', 'python-mysqldb'],
```

Comment:
I wouldn't force people to use some specific versions. Also I don't think it's needed to list all of these packages as most are delivered by dependencies. I believe only these are needed:
```mariadb-server, galera-3, mariadb-client, python-mysqldb```